firmware/library/L2_Utilities/util.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+
+namespace Utility
+{
+static uint32_t Random(uint32_t min, uint32_t max);
+static void SetRandomSeed();
+static uint32_t BitExtract(uint32_t extractee, uint8_t start_bit,
+                           uint8_t width);
+static uint32_t BitPlace(uint32_t placee, uint8_t s_bit, uint8_t bits,
```

Comment:
Do not short hand variable names. Spell out the whole word/phrase so it is readable.